themes-default/slim/views/home.mako

```diff
@@ -21,6 +21,100 @@ const startVue = () => {
             return {
                 header: 'Show List'
             };
+        },
+        mounted() {
+            $('#showListTableSeries:has(tbody tr), #showListTableAnime:has(tbody tr)').tablesorter({
```

Comment:
@sharkykh I tried using `nextTick` here to prevent the issue you're talking about but I could get get it working without `nextTick` so I don't know what todo now.

Reply:
`textExtraction` seems outdated for it anyway. See below.